velox/exec/tests/PrintPlanWithStatsTest.cpp

```diff
@@ -163,6 +163,7 @@ TEST_F(PrintPlanWithStatsTest, innerJoinWithTableScan) {
        {"        hashtable.capacity\\s+sum: 200, count: 1, min: 200, max: 200"},
        {"        hashtable.numDistinct\\s+sum: 100, count: 1, min: 100, max: 100"},
        {"        hashtable.numRehashes\\s+sum: 1, count: 1, min: 1, max: 1"},
+       {"        joinTableBuildWallNanos\\s+sum: .+, count: 1, min: .+, max: .+"},
```

Comment:
Looks like this name is not consistent with the rest: 

joinTableBuildWallNanos vs. hashtable.numDistinct